packages/elements/src/utils/duplicatePage.js

```diff
@@ -25,18 +21,15 @@ import createNewElement from './createNewElement';
 import duplicateElement from './duplicateElement';
 
 const duplicatePage = (oldPage) => {
-  // Remove title and postId for inserting the page.
+  // Remove title and templateId for inserting the page.
```

Comment:
I tried to test this part and can't see when would a template / post ID be assigned at all. Thought that perhaps when  inserting a page template and then duplicating it but doesn't seem like it. Were you able to test in which case we need to remove the title and ID when duplicating?

Reply:
Indeed, doesn't look like that happens at all.

I tried inserting default page templates and custom saved page templates, but there's no `templateId` leaking into this.

Looking at `handlePageClick`, it's specifically omitted:

https://github.com/GoogleForCreators/web-stories-wp/blob/7f3af11a2fc2bc0e35733d1e79462f39918b2101/packages/story-editor/src/components/library/panes/pageTemplates/templateList.js#L72-L76

So yeah, seems like it's safe to remove this comment & check.

For context, originally introduce in #7290 and then adjusted in #7318